app/models/manageiq/providers/openstack/inventory/persister/infra_manager.rb

```diff
@@ -1,16 +1,18 @@
 class ManageIQ::Providers::Openstack::Inventory::Persister::InfraManager < ManageIQ::Providers::Openstack::Inventory::Persister
-  include ManageIQ::Providers::Openstack::Inventory::Persister::Definitions::OrchestrationStackCollections
-
   def initialize_inventory_collections
-    add_collection(infra, :miq_templates) { |b| b.add_properties(:model_class => manager.class::Template) }
+    add_collection(infra, :miq_templates)
     add_collection(infra, :hardwares)
     add_collection(infra, :operating_systems)
-    add_collection(infra, :clusters) { |b| b.add_properties(:model_class => manager.class::Cluster) }
-    add_collection(infra, :orchestration_stacks)
+    add_collection(infra, :clusters)
     add_collection(infra, :hosts)
     add_collection(infra, :host_disks)
     add_collection(infra, :host_hardwares)
     add_collection(infra, :host_operating_systems)
-    add_orchestration_stack_collections
+    add_collection(infra, :orchestration_stacks)
+    add_collection(infra, :orchestration_stacks_resources)
+    add_collection(infra, :orchestration_stacks_outputs)
+    add_collection(infra, :orchestration_stacks_parameters)
+    add_collection(cloud, :orchestration_templates)
```

Comment:
TODO this doesn't work with `add_collection(infra, :orchestration_templates)` not sure why not yet